src/engine/engine.rs

```diff
@@ -52,7 +53,7 @@ pub trait KeyActions {
 
     /// Unset a key with the given key description in the root persistent kernel
     /// keyring.
-    fn unset(&mut self, key_desc: &KeyDescription) -> StratisResult<DeleteAction<Key>>;
+    fn unset(&mut self, key_desc: &KeyDescription) -> StratisResult<MappingDeleteAction<Key>>;
```

Comment:
This change is related to log messaging. We already had a `DeleteAction<Key>` for unsetting keys but we needed this type for deleting keyring bindings. This resulted in incorrect messages in our mutating action logging implementation. I figured that overloading the `DeleteAction` type did not make sense given that setting keys in the keyring returns a `MappingCreateAction` and it made the most sense to me to return something that matches that so that those types, when defined with the type parameter `Key`, are very specifically tied to keyring set/unset operations.

Reply:
ok!